lib/OpenQA/Controller/API/V1/Job.pm

```diff
@@ -206,7 +206,7 @@ sub restart {
     }
 
     my @res = OpenQA::Scheduler::job_restart($target);
-    $self->render(json => {result => \@res});
+    $self->render(json => {result => $res[0], test_url => $self->url_for('test', testid => $res[0]) });
```

Comment:
This does not handle the situation when more jobs are restarted at once, see :203